data.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Agumon', '2020-02-03', 0, FALSE, 10.23);
```

Comment:
- [ ] The value for the `nuetered` column for Agumon should be true, kindly fix this as it affects the results of the queries for User's

Screenshot 
![image](https://github.com/sam2if/Vet_clinic/assets/81932478/f1a5a28e-8d74-421a-8ffd-0afca69e272e)
- Also the values for escape attempts and weight_kg are swaped, kindly fix this entry